### PR TITLE
Add MetaWiki

### PR DESCRIPTION
**Pull request by RaikireHiuduo**
### Description

A FAQ of sort, for osu-wiki in GitHub detailing the quirks.

### Status

- In-question state (as in, do we like, need this?)

### Changelog

- Made the file.

### Checklist

- [ ] Do we need this?
- [ ] Better folder name.
- [ ] Make a better structure or write it properly (it is taken out from old owcg's FAQ section).

### CODE REVIEW

File: wiki/MetaWiki/en.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+Addition, deletion, moving, and renaming of media is easier now and requires no special wiki function.
+However, this too has to undergo the pull request protocol.
+While osu!wiki will resize images to the width size limit if the image is too large, specific media size requirements must be handled manually.
+
+The most damaging change for pages that use it liberally would be **no Templates support**.
+You must manually duplicate the template and paste it accordingly with proper changes to the article you are editing, _on every language file_.
+
+Tables are supported by a bare-minimum with GFM (without extensions), unlike MediaWiki.
+
+Markdown is sensitive, as using the wrong capitalization in links can break them (i.e. links and images).
```
Comment (sr229) [inline wiki/MetaWiki/en.md]: L126: Rephrase opening phrase. ``Markdown is sensitive`` => ``Markdown is case-sensitive``

be a bit verbose.

File: wiki/MetaWiki/en.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+You can make mass changes easily and efficiently, and you can delay your changes for as long as you like.
+
+Unlike the old osu!wiki, changes has to be made via pull requests which are then reviewed for errors and myths before merging by someone with write permission to the master branch.
+This adds a layer of security against spammers and vandals.
+
+Addition, deletion, moving, and renaming of media is easier now and requires no special wiki function.
+However, this too has to undergo the pull request protocol.
+While osu!wiki will resize images to the width size limit if the image is too large, specific media size requirements must be handled manually.
+
+The most damaging change for pages that use it liberally would be **no Templates support**.
```
Comment (sr229) [inline wiki/MetaWiki/en.md]: L121: Typo detected.

File: wiki/MetaWiki/en.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+You _could_ also make an issue in [osu-wiki's Issues tab][GH Issue] to get help; however, the response time will vary and likely to be ignored if it is off-topic or is a trivial issue.
+
+## What happened to the old osu!wiki?
+
+Deprecated due to known performance issues and no solid solution to prevent spam.
+
+## How would this differ from the old wiki (running on MediaWiki)?
+
+osu!wiki will be ran like a git repo (repository).
```
Comment (sr229) [inline wiki/MetaWiki/en.md]: L111: Fun fact: osu! wiki is ran as a repository, NOT like a repository.

**Comment by sr229** [comment]
this could be merged with the main FAQ but as its own article IMHO

CC @TPGPL for  feedback

**Comment by TPGPL** [comment]
Probably gonn merge it with FAQ

**Comment by Repflez** [comment]
>Do we need this?

I'm asking the same, _do we really need this thing?_ As @TPGPL said, this is more FAQ material than something on its own.

**Comment by RaikireHiuduo** [comment]
#553

**Pull request closed without merging**